pkg/exec/exec.go

```diff
@@ -33,9 +33,9 @@ func GetLogger(ctx context.Context) log.Logger {
 		return *logger
 	}
```

Comment:
Is this phrased more nicely as:

```go
logger, ok := ctx.Value(middleware.LoggerKey).(*log.Logger)
if !ok {
  log.Fatal("Unable to retrieve logger from context, log lines will be missing")
}

return logger
```

or perhaps even

```go
return ctx.Value(middleware.LoggerKey).(*log.Logger)
```

which will also panic on a bad logger, which should never happen, and would print a stack trace.

Reply:
Yes that makes a lot more sense! 🤦‍♂ Fixed